modules/installation-osp-kuryr-api-scaling.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-kuryr-octavia-configure{context}"]
+= Scaling API using {rh-openstack} Octavia
+
+API can be scaled on {product-title} cluster running on {rh-openstack} with
+Octavia support
+Octavia is a requirement to deploy {product-title} with Kuryr, as Kuryr uses it
+to create the services, but when installing without Kuryr it is not. So first
+step is to ensure Octavia is available.
+If it is, it can be leveraged to create extra APIs IPs and better scale with the
+load. This allows to remove the API bottleneck as currently the traffics enters
+through the master node that holds the API VIP.
+
+If {product-title} cluster was installed with Kuryr, the Cluster Network
+Operator already creates an internal Octavia Load Balancer (Amphora driver) used
+for by the pods that wants to reach the kubernetes service IP (e.g.,
+172.30.0.1). Note if the api-int DNS record is used, this load balancer is not
+used.
+
+As this loadbalancer already exits, we can simply associate the API Floating IP
+to it:
++
+----
+$ openstack floating ip unset $API_FIP
+$ openstack floating ip set  --port $(openstack loadbalancer show -c vip_port_id
+-f value ${OCP_CLUSTER}-kuryr-api-loadbalancer) $API_FIP
+----
+
+
+On the other hand, if {product-title} cluster was installed without kuryr, or a different load
+balancer(s) wants to be created, then the next steps can be used:
+
+. Create an Octavia Load Balancer (Amphora as it requires L7 capabilities). Note
+the subnet selected is the same where the worker node VMs are running.
```

Comment:
Probably, but as API was less of an issue than Ingress... In order to not be Octavia driver specific. Perhaps we can add a note about if ovn-octavia is available it can be used and you avoid the single point/VM issue